lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,22 @@
-
+require 'base64'
+require 'securerandom'
 
 desc "Generate a batch of invite codes"
-task :generate_invites, [:n] => :environment do |task, args|
+task :generate_invites, [:n, :u] => :environment do |task, args|
```

Comment:
This does not seem related to the pull request. It probably makes sense anyway - just making sure it's what you want.